multus/multus.go

```diff
@@ -294,6 +294,24 @@ func cmdDel(args *skel.CmdArgs, exec invoke.Exec, kubeClient k8s.KubeClient) err
 		return err
 	}
 
+	if args.Netns == "" {
+		return nil
+	}
+	netns, err := ns.GetNS(args.Netns)
+	if err != nil {
+		if err != nil {
```

Comment:
"if err != nil {" is duplicated and nested. How about to make it one?

Reply:
I don't know how it came it.. My bad